.appveyor.yml

```diff
@@ -38,7 +38,7 @@ for:
   build_script:
     - dotnet build -c Release
   test_script:
-    - bash scripts/test.sh --filter TestCategory!=RemotePcap
```

Comment:
lets put this commit first, before the send packet changes, so we keep the code buildable at each commit. It greatly helps with bisecting if we ever need to do that.

Reply:
Done